04_environment/index.tex

```diff
@@ -1 +1,400 @@
+\texttt{DisasterEffect} & Island's Proximity distance to the eye of the storm & -Island geography\newline -Eye of the storm location & Determine the islands damage \\ \hline
+\end{tabular}
+\caption{\label{tab:table-name}Disaster's main functions}
+\end{center}
+\end{table}
+
+The disaster has been computed as part of the infrastructure: Each island has a fixed position, characterised using Cartesian coordinates. They are located in the archipelago which captures the collection of island geographies including the bounding regions of the entire archipelago.\\
+
+Furthermore, to determine when a disaster is going to occur, we have used the Bernoulli random variable (probability of a disaster occurring is low).\\
+
+The Spatial probability distribution function "Spatial PDF" is uniform and it controls the distribution works of XY grid of the location of the disaster. The magnitude is then exponentially distributed (it exponentially more likely to have a smaller magnitude as we want large disasters to be more rare, occur less frequently). In addition, we have created a magnitude variable "Lambda" which can vary the slope of the exponential function (create a harsher environment when increasing the "lambda variable" by creating larger disasters and vice versa). In addition, the eye of the storm (disaster epicentre) is limited within the bounds of the environment (The environment bounds are fixed, they are specified by the users).\\
+
+\begin{figure}[!htb]
+    \centering
+    \includegraphics[width=1\textwidth]{04_environment/Images/Disaster output infrastructure.png}
+    \caption{Infrastructure Disaster outcome}
+    \label{Images:Disaster output infrastructure}
+\end{figure}
+
+The image above illustrates the output of the disaster, it first shows the magnitude of the disaster and the precise location of the eye of the storm, it indicates as well the fixed location of each island as well as the proportional effect of each island. For example, Island 2 with a fixed location of (5,0) has the largest disaster effect, which means it is the closest to the eye of the storm.\\
+
+\subsection{Future works Concepts}
+
+The disaster is a random function, meaning that it is possible to predict it with the current game.\\
+
+There are multiple future works ideas that would be developed in the design aspect such as; a deterministic disaster which would be  designed as a straight line that accumulates during the days, once the threshold has been met then the disaster would occur. Another deterministic disaster idea would be to link the disaster’s magnitude to time. Thus, agents will have to learn from past disaster occurrences, creating a memory. Agents will start learning from past disasters and will be able to forecast the next one. Therefore, the agents will be able to forecast that if a disaster did not surface for a long period of time, then the magnitude of the next disaster will be much larger than the previous one.\\
```

Comment:
bad sentence (the first two) -- revise

Reply:
same with unnecessary \\ at the end - does not belong at the end of every paragraph